web/ci.sh

```diff
@@ -50,11 +50,14 @@ function web_sentry_upload () {
     ARTIFACT_FOLDER="$KEYMAN_ROOT/web/build/publish/release/"
   fi
 
-  pushd "$ARTIFACT_FOLDER"
-  echo "Uploading to Sentry..."
-  sentry-cli releases files "$VERSION_GIT_TAG" upload-sourcemaps --strip-common-prefix "@keymanapp/keyman/" --rewrite --ext js --ext map --ext ts || fail "Sentry upload failed."
+  echo "Uploading $(builder_term app/$1) ($ARTIFACT_FOLDER) to Sentry..."
+
+  # The "$ARTIFACT_FOLDER" bit is being used as the path to the sourcemaps.
+  # --strip-common-prefix does not take an argument, unlike --strip-prefix.  It auto-detects
+  # the most common prefix instead.
+  sentry-cli releases files "$VERSION_GIT_TAG" upload-sourcemaps --strip-common-prefix "$ARTIFACT_FOLDER" \
```

Comment:
So, the big mistake I made:  `--strip-common-prefix` doesn't take a parameter.

After drilling down heavily into the `sentry-cli` tool's `--help` sections:
```
        --strip-common-prefix
            Similar to --strip-prefix but strips the most common prefix on all sources references.

        --strip-prefix <PREFIX>
            Strips the given prefix from all sources references inside the upload sourcemaps (paths
            used within the sourcemap content, to map minified code to it's original source). Only
            sources that start with the given prefix will be stripped.
            This will not modify the uploaded sources paths. To do that, point the upload or upload-
            sourcemaps command to a more precise directory instead.
```

See, I thought I needed to tell is what the prefix was.  I was wrong.  Since the _original_ original was also run through `npm run`, the path it had been providing was needed to actually point the cli tool to the files to upload.

(Reminder of code section)

https://github.com/keymanapp/keyman/blob/7a67894bbbdfe53b0da1c85dfc5b3d9a1965c74d/web/source/build.sh#L553-L560

While there's a `pushd` and `popd` in the old block... they didn't actually affect the `sentry-cli` tool at all.  I'm pretty sure part of the cause behind my bad inference was reading too much into their presence.  I still should have double-checked the build logs before merging, though.